src/CommonLib/Processors/ACLProcessor.cs

```diff
@@ -41,7 +42,29 @@ static ACLProcessor() {
             };
         }
 
-        public ACLProcessor(ILdapUtils utils, ILogger log = null) {
+        /// Represents a lightweight Access Control Entry (ACE) used to compute hash values
+        /// for AdminSDHolder purposes
+        internal class ACEForHashing {
```

Comment:
suggestion (non-blocking): This class definition might be better placed at the end of the file, or at least after all the constructors for ACLProcessor have been defined.